test/sumoLogger.test.js

```diff
@@ -216,6 +216,27 @@ describe('sumoLogger', () => {
             );
         });
 
+        it('should send log message exactly as provided if raw option enabled', () => {
+            const logger = new SumoLogger({
+                endpoint,
+                raw: true
+            });
+            
+            logger.log([message], {
```

Comment:
Could you add a test case for when the raw text message is not sent in an array please?

Reply:
Thanks for the quick response @clementallen, I've added this test.